css-highlight-api-1/Overview.bs

```diff
@@ -308,13 +308,15 @@ Painting</h4>
 					</style>
 					<body>Lorem Ipsum.
 					<script>
+						let text_node = document.body.firstChild;
```

Comment:
I think we should use `textNode` instead of `text_node`. That seems to be more aligned with the style I've seen in other specs.

Reply:
Makes sense, there aren't any other long variable names in this draft and I thought it was a spec-independent thing, but your comment is right, I'll change this.